chapter_attention-mechanisms/attention-cues.md

```diff
@@ -200,10 +211,15 @@ we consider a simple case where
 the attention weight is one only when the query and the key are the same; otherwise it is zero.
 
 ```{.python .input}
-#@tab all
+#@tab mxnet, pytorch
 attention_weights = d2l.eye(10).reshape(1, 1, 10, 10)
 show_heatmaps(attention_weights, xlabel='Keys', ylabel='Queries')
 ```
+```{.python .input}
+#@tab tensorflow
+attention_weights = d2l.reshape(d2l.eye(10), (1, 1, 10, 10))
+show_heatmaps(attention_weights, xlabel='Keys', ylabel='Queries')
+```
```

Comment:
I think all of this can be clubbed into one tab with `#@tab all`
Just use the following code

#@tab all
attention_weights = d2l.reshape(d2l.eye(10), (1, 1, 10, 10))
show_heatmaps(attention_weights, xlabel='Keys', ylabel='Queries')

Reply:
Okay

Reply:
Its showing error for the mxnet code.
keeping same for pytorch & tf , while changing for mxnet to original one.
MXNetError: MXNetError: Invalid Parameter format for newshape expect Shape(tuple) but value='([1, 1, 10, 10],)', in operator _np_reshape(name="", order="C", newshape="([1, 1, 10, 10],)")

Reply:
keeping all 3 frameworks unchanged due to the mxnet error when using #@tab all